pegasus/sites.v3/code.org/public/yourschool/about.md

```diff
@@ -21,7 +21,9 @@ We started with the definition of computer science developed by the [Computer Sc
 
 _Computer science is the study of computers and algorithms, including their principles, their hardware and software designs, their implementation, and their impact on society._
 
-Learning computer science means learning how to
create new technologies, rather than just using
them. 
```

Comment:
I didn't touch this line, I think this is just a bug with Git processing the markdown since `^M` represents a newline and it's just being processed on the right